### PR TITLE
Fix-649

### PR DESCRIPTION
**Pull request by chestozo**

### CODE REVIEW

File: src/ns.update.js [hunk 1]
```diff
@@ -321,6 +321,10 @@
             this.view._getUpdateTree(tree);
         }
 
+        if (tree.__hasInvalidModels__) {
+            this.abort();
+        }
```
Comment (chestozo) [inline src/ns.update.js]: Сломалось где-то 5 тестов.
К примеру, тут https://github.com/yandex-ui/noscript/blob/master/test/spec/ns.update.edge-cases.js#L228

Похоже, прямо весь ns.Update отменять из-за одного вида, у которого invalid модель - не самая лучшая идея.

@vitkarpov

**Comment by chestozo** [comment]
Это решение отмели - сделал новый заход #651
